src/CheckIn.js

```diff
@@ -599,6 +599,7 @@ class CheckIn extends React.Component {
                     component={Select}
                     dataOptions={actionOptions}
                     data-test-action
+                    isEqual={(a, b) => true}
```

Comment:
As I understand it, we can simplify to
`isEqual={() => true}`
Could you please double check?